Content.Shared/Storage/EntitySystems/SharedEntityStorageSystem.cs

```diff
@@ -344,6 +344,14 @@ public bool TryCloseStorage(EntityUid target)
         return true;
     }
 
+    public bool IsOpen(EntityUid target, SharedEntityStorageComponent? component = null)
```

Comment:
Can't you just do a `TryComp` and check `component.Open` where you need it?

Reply:
The issue is that in shared I can't do a TryComp for SharedStorageComp, I get an error for it. But when it gets resolved like this, it works fine.